# 1.   Introdução






A prematuridade, caracterizada pelo nascimento de um bebê antes de completar 37 semanas de gestação, é um dos desafios mais significativos enfrentados pela medicina neonatal e pela saúde pública em todo o mundo. O aumento da taxa de nascimentos prematuros nas últimas décadas tem suscitado crescente preocupação, uma vez que esses bebês podem enfrentar uma série de problemas médicos e desenvolvimentais, bem como desafios emocionais para suas famílias.

Os bebês prematuros são especialmente vulneráveis devido à imaturidade de seus órgãos e sistemas, que normalmente continuariam a se desenvolver dentro do útero da mãe. Essa imaturidade pode levar a complicações de curto e longo prazo, afetando não apenas o início da vida, mas também repercutindo na infância, adolescência e até mesmo na vida adulta.

Dentre os problemas imediatos associados à prematuridade, destacam-se a dificuldade respiratória, a susceptibilidade a infecções, a instabilidade térmica e a alimentação desafiadora. Além disso, bebês prematuros têm maior probabilidade de desenvolver complicações neurológicas, como hemorragias intracranianas e distúrbios do desenvolvimento cerebral, que podem afetar seu crescimento e aprendizado.

Alguns estudos e programas foram desenvolvidos para combater os efeitos da prematuridade, entre eles, podemos citar o  Programa de Saúde e Desenvolvimento do Bebê (IHDP), que foi um estudo sobre intervenção em bebês com baixo peso ao nascer (BPN) realizado na década de 1980 - representa um marco significativo. Trata-se de uma iniciativa pioneira, sendo o primeiro experimento clínico randomizado conduzido em diversos locais, com o objetivo de avaliar a eficácia da combinação de serviços destinados ao desenvolvimento infantil e apoio familiar, juntamente com acompanhamento pediátrico, visando reduzir problemas de desenvolvimento, comportamentais e de saúde em prematuros BPN. (Ross, 2008).

Este trabalho tem como objetivo explorar de forma mais aprofundada os problemas associados à prematuridade em bebês, analisando suas possíveis causas.



---






# 2.   Referencial Teórico






De acordo com estimativas, ocorre anualmente em todo o mundo o nascimento de 15 milhões de bebês prematuros (Who, 2018). O Brasil, por sua vez, ocupa o 9º lugar em números absolutos de nascimentos prematuros, tornando essa questão um relevante problema de saúde pública (Blencowe et.al., 2012) . Conforme a definição da Organização Mundial da Saúde (OMS), são considerados neonatos prematuros aqueles que nascem antes de completar as 37 semanas de gestação. Com base na idade gestacional, esses recém-nascidos podem ser classificados ainda em subcategorias, tais como pré-termos extremos (até 28 semanas incompletas), muito prematuros (de 28 semanas completas a 32 semanas incompletas) e prematuros moderados a tardios (de 32 semanas incompletas a 37 semanas incompletas)(Who, 2018). Essa classificação é de extrema importância para avaliar o grau de prematuridade e auxiliar no planejamento de cuidados adequados a cada grupo de bebês prematuros, visto que essas crianças têm necessidades específicas de acordo com o estágio de desenvolvimento em que se encontram ao nascer. Diante dessa significativa incidência de prematuridade, torna-se imperativo que medidas de prevenção e políticas de saúde sejam adotadas com o objetivo de reduzir os riscos associados a essa condição e garantir o melhor prognóstico possível para esses recém-nascidos, promovendo assim a saúde materno-infantil e o bem-estar global da população.

Ramos e Cuman (2009), buscaram identificar o perfil das mães e dos bebês prematuros nascidos vivos, assim como caracterizar os recém-nascidos prematuros em situação de risco para o crescimento e desenvolvimento. A pesquisa epidemiológica de corte transversal foi conduzida na cidade de Guarapuava, no estado do Paraná. Os dados foram obtidos por meio do sistema de informações sobre nascidos vivos (SINASC), com base nas declarações de nascidos vivos. A análise estatística foi realizada em uma amostra composta por 106 declarações de nascidos vivos prematuros, no período de janeiro a junho de 2005, juntamente com informações sobre suas respectivas mães.

Foram consideradas as seguintes variáveis para análise: características sociodemográficas, condições da gestação e do parto, bem como as características dos prematuros nascidos vivos. Os resultados obtidos permitiram concluir que conhecer e avaliar o perfil das mães, assim como o número e a situação dos nascimentos de bebês prematuros em uma determinada área e período de tempo, é de suma importância para a identificação de riscos vitais relacionados às condições do nascimento, crescimento e desenvolvimento infantil.

Os autores concluíram que o perfil das mães dos bebês prematuros e a caracterização dos nascidos vivos são diretamente influenciados pelas condições socioeconômicas e sanitárias do local onde ocorre a gestação e o parto. Essas condições, por sua vez, têm um impacto significativo na qualidade de vida futura dessas crianças, tornando-se um fator contribuinte para os altos índices de mortalidade infantil, que é um dos principais problemas observados no município estudado.

No que condiz aos fatores de riscos, podemos destacar os seguintes estudos. FERREIRA JUNIOR et al., (2018) mostrou que foi observada a predominância do sexo masculino (53,3%) entre os nascimentos prematuros.

O estudo conduzido por Kawakita, Wilson, Grantz, Landy, Huang e Gomez-Lobo (2016) investigou uma amostra de 43.537 mulheres com idade inferior a 25 anos nos Estados Unidos, com o objetivo de analisar a relação entre a idade materna e o risco de parto prematuro.

Os resultados da pesquisa demonstraram que existe uma associação significativa entre a idade materna mais jovem e um aumento nas chances de ocorrer parto prematuro. Em outras palavras, quanto mais jovem a mulher, maior é a probabilidade de ela dar à luz um bebê prematuro.

A idade materna é um fator reconhecido como condicionante de risco durante a gravidez, especialmente quando ocorre fora da faixa etária considerada ideal para a gestação, ou seja, nos extremos da vida reprodutiva, abrangendo tanto as gestações precoces quanto as tardias (Cabral; Santos & Cano, 2017).

Em mulheres com gestação tardia, tem sido observado um aumento no número de abortos espontâneos e induzidos, bem como um maior risco para a mortalidade perinatal, baixo peso ao nascer, parto pré-termo e recém-nascidos com baixo peso em relação à idade gestacional. As gestações em mulheres de idade materna avançada também são consideradas de alto risco, devido, principalmente, à crescente incidência de síndromes hipertensivas, ruptura prematura de membranas e presença de diabetes gestacional. Além disso, existe uma maior chance de o Índice de Apgar no quinto minuto ser inferior a sete, o que pode indicar a ocorrência de anomalias cromossômicas, macrossomias (bebês com peso acima do esperado para a idade gestacional), baixo peso ao nascer e menor vitalidade do neonato. Sendo assim, um escore menor que 7 no Índice de Apgar é considerado um sinal de alerta para a necessidade de uma atenção especial e cuidados adicionais ao recém-nascido (Silva; Cavalcante; Carneiro & Oliveira, 2020).

É fundamental que o uso de tabaco, bebidas alcoólicas e drogas ilícitas seja suspenso durante a gestação, porém, muitas vezes, isso não ocorre. Um estudo realizado com 61 gestantes revelou que a maioria das mulheres que eram fumantes e ex-fumantes não tinha conhecimento dos efeitos prejudiciais que o tabagismo pode causar ao feto e ao recém-nascido, nem das doenças relacionadas a essa prática (Bertani; Garcia; Tanni & Godoy, 2015).

Diante dessa realidade, torna-se essencial que os profissionais da área da saúde ofereçam orientação adequada sobre os possíveis riscos associados ao tabagismo, assim como os perigos da exposição à radiação durante a gestação. Além disso, é importante alertar sobre os malefícios do consumo de bebidas alcoólicas durante a gravidez, pois o álcool pode levar a abortamento, restrição do crescimento fetal, retardo mental e malformações no bebê.

O retardo mental é considerado uma das principais complicações que o álcool pode provocar no recém-nascido, e é fundamental destacar que essa condição pode ser totalmente prevenida (Sociedade de Pediatria de São Paulo, 2017).

Promover a conscientização sobre essas questões é fundamental para proteger a saúde da mãe e do bebê durante a gestação, garantindo uma gravidez mais saudável e segura. O papel dos profissionais de saúde é crucial nesse sentido, pois sua orientação e suporte podem ajudar as gestantes a tomarem decisões mais saudáveis para si mesmas e para seus filhos.

No que diz respeito à renda familiar mensal, o estudo realizado por Almeida, et al (2012), com 58 puérperas de bebês prematuros evidenciou que 89% delas tinham uma renda mensal de menos de dois salários-mínimos. Essa variável, isoladamente ou em conjunto com outros fatores, foi identificada como um fator que aumenta o risco de partos prematuros.

Conforme amplamente documentado na literatura, a baixa escolaridade materna está associada a nascimentos prematuros e pode ser influenciada pelo menor acesso à informação e aos serviços de saúde. O estudo de Oliveira, et al.(2016) destaca que mulheres com escolaridade inadequada para a idade têm duas vezes mais chances de terem partos prematuros. Além disso, Bernardino, et al. (2019) também observou uma prevalência de mães de bebês prematuros com apenas o ensino médio completo ou ainda incompleto.

O acompanhamento pré-natal é fundamental para identificar e investigar situações de risco durante a gestação, prevenindo desfechos indesejáveis. No estudo de Almeida et al. (2012), com 58 mães de bebês prematuros, cerca de 60% realizaram menos de cinco consultas pré-natal, apresentando cinco vezes mais chances de prematuridade em comparação com as mães de bebês a termo. Outros estudos, como os de Oliveira et al. (2016), também relacionaram o número inadequado de consultas pré-natal com o parto prematuro.

Esses resultados enfatizam a importância da atenção à saúde materna durante a gravidez, especialmente para as mulheres que apresentam baixa renda, baixa escolaridade e acesso limitado aos cuidados pré-natais. O fortalecimento do acompanhamento pré-natal adequado pode contribuir para reduzir os riscos de partos prematuros e melhorar os resultados de saúde tanto para a mãe quanto para o bebê. Ações que visem melhorar o acesso a informações e serviços de saúde, bem como programas de conscientização sobre a importância do pré-natal, são essenciais para abordar esses fatores de risco e promover uma gravidez mais segura e saudável.



# 3.   Resultados e Discussão






Para analisar a variável prematuridade, sera adotada uma extratificação de acordo com o proposto pela OMS, no qual ela considera a prematuridade acima de 9 semanas, extrema, entre 9 e 6 semanas, alta, entre 6 a 1 semana, moderada e nenhuma prematuridade como ausente, contudo será analisado mais uma extratificação, quando o nascimento é póstermo, ou seja, quando os dados estiverem menores que 0.

De acordo, com a revisão bibliográfica, e com os dados disponibilizados pelo IHDP, será analisado as possíveis causas para a prematuridade, como indicadores de uso de tabaco, alcool durante a gravidez, como tambem a quantidade de pre-natais realizadas, o peso dos recém-nascidos, a renda da família, a etnia da mãe, o QI do bebê aos 3 anos de idade, entre outras variáveis disponíveis. Essa análise será necessária para entender o que pode ter levado o nascimento prematuro, como também suas consequências, QI dos RCN.

Para realizar essa análise foram elaborados algumas hipóteses baseados na revisão bibliográfica:


1.  A primeira hipótese foi que Mulheres Jovens (A baixo de 20 anos), principalmenete de etnia negra (devido ao contexto do Estado norte americano) teriam maiores chances de apresentarem uma prematuridade.  
2.  A segunda hipótese, também será relacionado entre idade e etnia, contudo agora avaliando Mulheres que tenham uma idade maior que 20 anos que são de etnias negras apresentem uma maior prematuridade.
3.  A Terceira hipótese se dará no efeito do Cigarro na Prematuridade, como também avaliando as consequências no desenvolvimento intelectual da criança.
4.  A quarta hipótese se dará na análise do efeito do Álcool na prematuridade, como também avaliando as suas consequências no desenvolvimento intelectual da criança.
5.  Na quinta hipótese será que Mulheres abaixo  de 20 anos e que não fizeram um prenatal apresentem maiores chances de terem bebês prematuros.
6.  Na sexta hipótese será que Mulheres acima  de 20 anos e que não fizeram um prenatal apresentem maiores chances de terem bebês prematuros.
7.  Na sétima hipótese será que mulheres de etnias negras ou hispânicas que não fizeram o prenatal apresentem maiores chances de apresentarem filhos prematuros.
8.  Na oitava hipótese será que Mulheres acima de 20 anos e que seja o primeiro filho tenham maiores chances de terem um bebê prematuro.
9.  Por fim, na Nona hipótese, que pessoas de rendas baixas, apresentem maiores chances de terem bebês prematuros e de baixo peso.

A análise será utilizando o R, com a plataforma do Google Colabs.

O código abaixo mostra o passo a passo da análise realizada neste trabalho. Também serão analisados os outiliers encontrados durante o estudo.

## Instalando e carregando os pacotes necessários

In [ ]:
# Procedimento usual na instalação de pacotes
install.packages(
    pkgs = 'pacman',
    lib = '/usr/local/lib/R/site-library'
)

In [ ]:
# Procedimento usual no carregamento de pacotes já instalados
library(package = 'pacman')

In [ ]:
# Usando o pacote 'pacman', a função 'p_load' instala, caso não esteja
# previamente, e carrega automaticamente os pacotes definidos
pacman::p_load('ggplot2','gridExtra')

Installing package into ‘/usr/local/lib/R/site-library’
(as ‘lib’ is unspecified)


gridExtra installed



## Carregando o arquivo

In [ ]:
# Retorna a pasta atual de trabalho do Colab
getwd()

[1] "/content"

In [ ]:
# Download de arquivos do Google Drive.
# Link Google drive: https://drive.google.com/file/d/1U1eVN0vvRBLHA1dXnjfBdfvGPB8jdyk_/view?usp=share_link
system('gdown --id 1U1eVN0vvRBLHA1dXnjfBdfvGPB8jdyk_')

In [ ]:
# Carrega o arquivos 'csv' separados por ';' e variáveis (colunas) do tipo "character"
IHDP <- read.csv(
    file = '/content/ihdp.csv',
    sep = ';',
    colClasses = 'character'
)

In [ ]:
# Retorna os primeiros elementos de um vetor, matriz, tabela ou dataframe.
head(x=IHDP, n=4)

idade etnia  edu              trab prenatal cigarro alcool genero    primog
1 33    branca superior         sim  sim      nao     nao    masculino nao   
2 22    negra  medio_incompleto nao  sim      nao     sim    masculino nao   
3 13    negra  medio_incompleto nao  sim      nao     nao    masculino sim   
4 25    negra  superior         sim  sim      nao     nao    masculino sim   
  peso premat hosp renda trt  QI 
1 1559 10     31   42500 caso 111
2 2240 3      4    5000  caso 81 
3 1900 6      9    12500 caso 92 
4 1550 8      50   42500 caso 103

## Preparando os dados para análises

In [ ]:
# Exibe as informações básicas e exemplos no dataframe
str(IHDP)

'data.frame':	4381 obs. of  15 variables:
 $ idade   : chr  "33" "22" "13" "25" ...
 $ etnia   : chr  "branca" "negra" "negra" "negra" ...
 $ edu     : chr  "superior" "medio_incompleto" "medio_incompleto" "superior" ...
 $ trab    : chr  "sim" "nao" "nao" "sim" ...
 $ prenatal: chr  "sim" "sim" "sim" "sim" ...
 $ cigarro : chr  "nao" "nao" "nao" "nao" ...
 $ alcool  : chr  "nao" "sim" "nao" "nao" ...
 $ genero  : chr  "masculino" "masculino" "masculino" "masculino" ...
 $ primog  : chr  "nao" "nao" "sim" "sim" ...
 $ peso    : chr  "1559" "2240" "1900" "1550" ...
 $ premat  : chr  "10" "3" "6" "8" ...
 $ hosp    : chr  "31" "4" "9" "50" ...
 $ renda   : chr  "42500" "5000" "12500" "42500" ...
 $ trt     : chr  "caso" "caso" "caso" "caso" ...
 $ QI      : chr  "111" "81" "92" "103" ...


In [ ]:
# Nome das colunas em maiúsculo com a função toupper.
colnames(IHDP) <- toupper(colnames(IHDP))
head(IHDP,3)

IDADE ETNIA  EDU              TRAB PRENATAL CIGARRO ALCOOL GENERO    PRIMOG
1 33    branca superior         sim  sim      nao     nao    masculino nao   
2 22    negra  medio_incompleto nao  sim      nao     sim    masculino nao   
3 13    negra  medio_incompleto nao  sim      nao     nao    masculino sim   
  PESO PREMAT HOSP RENDA TRT  QI 
1 1559 10     31   42500 caso 111
2 2240 3      4    5000  caso 81 
3 1900 6      9    12500 caso 92

In [ ]:
# Conversão das colunas 'idade','peso','premat','hosp','renda' e 'QI'
# para variáveis numéricas e inteiras

# Vetor de colunas a serem convertidas em inteiro
col_int <- c('IDADE','PESO','PREMAT','HOSP','RENDA','QI')

# Iterar coluna a coluna e realizar a conversão
for (col in col_int) {
    IHDP[,col] <- as.integer(IHDP[,col])
}

# Retornar o dataframe resultante
str(IHDP)

'data.frame':	4381 obs. of  15 variables:
 $ IDADE   : int  33 22 13 25 19 19 26 20 23 28 ...
 $ ETNIA   : chr  "branca" "negra" "negra" "negra" ...
 $ EDU     : chr  "superior" "medio_incompleto" "medio_incompleto" "superior" ...
 $ TRAB    : chr  "sim" "nao" "nao" "sim" ...
 $ PRENATAL: chr  "sim" "sim" "sim" "sim" ...
 $ CIGARRO : chr  "nao" "nao" "nao" "nao" ...
 $ ALCOOL  : chr  "nao" "sim" "nao" "nao" ...
 $ GENERO  : chr  "masculino" "masculino" "masculino" "masculino" ...
 $ PRIMOG  : chr  "nao" "nao" "sim" "sim" ...
 $ PESO    : int  1559 2240 1900 1550 2270 1550 2330 2410 1776 2140 ...
 $ PREMAT  : int  10 3 6 8 5 4 9 3 6 5 ...
 $ HOSP    : int  31 4 9 50 4 13 8 6 30 2 ...
 $ RENDA   : int  42500 5000 12500 42500 5000 12500 17500 11344 30000 8750 ...
 $ TRT     : chr  "caso" "caso" "caso" "caso" ...
 $ QI      : int  111 81 92 103 81 94 104 82 107 79 ...


In [ ]:
# Conversão das colunas restantes do tipo caractere para variáveis categóricas
# conhecidos como fatores no R.
cols <- colnames(IHDP)
for (col in cols) {
    if (is.character(IHDP[,col])) {
        IHDP[,col] <- as.factor(IHDP[,col])
    }
}

str(IHDP)

'data.frame':	4381 obs. of  15 variables:
 $ IDADE   : int  33 22 13 25 19 19 26 20 23 28 ...
 $ ETNIA   : Factor w/ 3 levels "branca","hispanica",..: 1 3 3 3 3 3 1 1 1 3 ...
 $ EDU     : Factor w/ 4 levels "medio","medio_incompleto",..: 3 2 2 3 2 1 2 2 1 1 ...
 $ TRAB    : Factor w/ 2 levels "nao","sim": 2 1 1 2 1 2 2 1 2 2 ...
 $ PRENATAL: Factor w/ 2 levels "nao","sim": 2 2 2 2 2 2 2 2 2 2 ...
 $ CIGARRO : Factor w/ 2 levels "nao","sim": 1 1 1 1 2 2 2 2 1 2 ...
 $ ALCOOL  : Factor w/ 2 levels "nao","sim": 1 2 1 1 1 2 1 2 1 1 ...
 $ GENERO  : Factor w/ 2 levels "feminino","masculino": 2 2 2 2 2 1 2 1 1 2 ...
 $ PRIMOG  : Factor w/ 2 levels "nao","sim": 1 1 2 2 1 2 1 1 1 1 ...
 $ PESO    : int  1559 2240 1900 1550 2270 1550 2330 2410 1776 2140 ...
 $ PREMAT  : int  10 3 6 8 5 4 9 3 6 5 ...
 $ HOSP    : int  31 4 9 50 4 13 8 6 30 2 ...
 $ RENDA   : int  42500 5000 12500 42500 5000 12500 17500 11344 30000 8750 ...
 $ TRT     : Factor w/ 2 levels "caso","controle": 1 1 1 1 1 1 1 1 1 1 ..

In [ ]:
# Alterando os níveis (categorias) da coluna 'GENERO'
levels(IHDP$GENERO) <- c('F','M')
levels(IHDP$GENERO)
str(IHDP)

[1] "F" "M"

'data.frame':	4381 obs. of  15 variables:
 $ IDADE   : int  33 22 13 25 19 19 26 20 23 28 ...
 $ ETNIA   : Factor w/ 3 levels "branca","hispanica",..: 1 3 3 3 3 3 1 1 1 3 ...
 $ EDU     : Factor w/ 4 levels "medio","medio_incompleto",..: 3 2 2 3 2 1 2 2 1 1 ...
 $ TRAB    : Factor w/ 2 levels "nao","sim": 2 1 1 2 1 2 2 1 2 2 ...
 $ PRENATAL: Factor w/ 2 levels "nao","sim": 2 2 2 2 2 2 2 2 2 2 ...
 $ CIGARRO : Factor w/ 2 levels "nao","sim": 1 1 1 1 2 2 2 2 1 2 ...
 $ ALCOOL  : Factor w/ 2 levels "nao","sim": 1 2 1 1 1 2 1 2 1 1 ...
 $ GENERO  : Factor w/ 2 levels "F","M": 2 2 2 2 2 1 2 1 1 2 ...
 $ PRIMOG  : Factor w/ 2 levels "nao","sim": 1 1 2 2 1 2 1 1 1 1 ...
 $ PESO    : int  1559 2240 1900 1550 2270 1550 2330 2410 1776 2140 ...
 $ PREMAT  : int  10 3 6 8 5 4 9 3 6 5 ...
 $ HOSP    : int  31 4 9 50 4 13 8 6 30 2 ...
 $ RENDA   : int  42500 5000 12500 42500 5000 12500 17500 11344 30000 8750 ...
 $ TRT     : Factor w/ 2 levels "caso","controle": 1 1 1 1 1 1 1 1 1 1 ...
 $ QI      : 

## Estratificações das variáveis

In [ ]:
# Exibe as informações estatísticas básica da variável 'PREMATURIDADE'
summary(IHDP$PREMAT)

   Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
 -7.000   1.000   1.000   1.493   2.000  14.000 

In [ ]:
# Estratificacao da  'PREMATURIDADE' dos filhos
labels <- c('postermo', 'ausente', 'moderada', 'alta', 'extrema')
breaks <- c(-Inf, 0, 1, 6, 9, Inf)
PREMATCAT <- cut(
    IHDP[,'PREMAT'],
    breaks,
    labels,
    right = FALSE
   )
IHDP$PREMATCAT <- PREMATCAT # IHDP[,'prematCAT'] <- prematCAT
table(IHDP$PREMATCAT)

#contagem dos dados da extratificacao
nrow(IHDP)
extrema <- subset(IHDP, PREMAT >=9)
nrow(extrema)
alta <- subset(IHDP, PREMAT >= 6 & PREMAT < 9 )
nrow(alta)
moderada <- subset(IHDP, PREMAT >= 1 & PREMAT < 6 )
nrow(moderada)
ausente <- subset(IHDP, PREMAT == 0)
nrow(ausente)
postermo <- subset (IHDP, PREMAT < 0)
nrow(postermo)


postermo  ausente moderada     alta  extrema 
     707      200     3165      242       67 

[1] 4381

[1] 67

[1] 242

[1] 3165

[1] 200

[1] 707

In [ ]:
# Estratificação de 'PESO'
labels <- c('baixo','normal')
breaks <- c(-Inf, 2500, Inf)
pesoCAT <- cut(
    IHDP[,'PESO'],
    breaks,
    labels,
    right = T
   )
IHDP$pesoCAT <- pesoCAT # IHDP[,'pesoCAT'] <- pesoCAT
table(IHDP$pesoCAT)

# Criando variáveis para conferir os valores da estratificação
pesoBaixo <- subset(IHDP, PESO  <= 2500)
pesoNormal <- subset(IHDP, PESO > 2500)
nrow (pesoBaixo)
nrow (pesoNormal)


 baixo normal 
   590   3791 

[1] 590

[1] 3791

In [ ]:
# Estratificação de 'RENDA'
labels <- c('baixa', 'media', 'alta')
breaks <- c(-Inf, 50000, 150000, Inf)
rendaCAT <- cut(
    IHDP[,'RENDA'],
    breaks,
    labels,
    right = T
   )
IHDP$rendaCAT <- rendaCAT # IHDP[,'rendaCAT'] <- rendaCAT
table(IHDP$rendaCAT)

# Criando variáveis para conferir os valores da estratificação
rendaBaixa <- subset(IHDP, RENDA  <= 50000)
rendaMedia <- subset(IHDP, RENDA > 50000 & RENDA <= 150000)
rendaAlta <- subset(IHDP, RENDA > 150000)
nrow (rendaBaixa)
nrow (rendaMedia)
nrow (rendaAlta)


baixa media  alta 
 3974   346    61 

[1] 3974

[1] 346

[1] 61

In [ ]:
# Na analise vai ser necessario fazer uma estratificacao da variavel QI
labels <- c('baixa', 'media', 'alta')
breaks <- c(-Inf, 90, 110, Inf)
qiCAT <- cut(
    IHDP[,'QI'],
    breaks,
    labels,
    right = TRUE
   )
IHDP$qiCAT <- qiCAT

## Divisão entre os grupos de caso e controle

In [ ]:
# Dataframe de pacientes do grupo caso
CASO <- subset(IHDP, TRT=='caso')
str(CASO)

'data.frame':	290 obs. of  19 variables:
 $ IDADE    : int  33 22 13 25 19 19 26 20 23 28 ...
 $ ETNIA    : Factor w/ 3 levels "branca","hispanica",..: 1 3 3 3 3 3 1 1 1 3 ...
 $ EDU      : Factor w/ 4 levels "medio","medio_incompleto",..: 3 2 2 3 2 1 2 2 1 1 ...
 $ TRAB     : Factor w/ 2 levels "nao","sim": 2 1 1 2 1 2 2 1 2 2 ...
 $ PRENATAL : Factor w/ 2 levels "nao","sim": 2 2 2 2 2 2 2 2 2 2 ...
 $ CIGARRO  : Factor w/ 2 levels "nao","sim": 1 1 1 1 2 2 2 2 1 2 ...
 $ ALCOOL   : Factor w/ 2 levels "nao","sim": 1 2 1 1 1 2 1 2 1 1 ...
 $ GENERO   : Factor w/ 2 levels "F","M": 2 2 2 2 2 1 2 1 1 2 ...
 $ PRIMOG   : Factor w/ 2 levels "nao","sim": 1 1 2 2 1 2 1 1 1 1 ...
 $ PESO     : int  1559 2240 1900 1550 2270 1550 2330 2410 1776 2140 ...
 $ PREMAT   : int  10 3 6 8 5 4 9 3 6 5 ...
 $ HOSP     : int  31 4 9 50 4 13 8 6 30 2 ...
 $ RENDA    : int  42500 5000 12500 42500 5000 12500 17500 11344 30000 8750 ...
 $ TRT      : Factor w/ 2 levels "caso","controle": 1 1 1 1 1 1 1 1 1 1 ...


In [ ]:
# Dataframe de pacientes do grupo controle
CTRL <- subset(IHDP, TRT=='controle')
str(CTRL)

'data.frame':	4091 obs. of  19 variables:
 $ IDADE    : int  19 22 24 24 27 27 24 28 21 27 ...
 $ ETNIA    : Factor w/ 3 levels "branca","hispanica",..: 1 1 1 1 1 1 1 1 1 1 ...
 $ EDU      : Factor w/ 4 levels "medio","medio_incompleto",..: 2 2 2 1 2 3 1 1 1 1 ...
 $ TRAB     : Factor w/ 2 levels "nao","sim": 2 1 1 2 2 1 2 2 2 2 ...
 $ PRENATAL : Factor w/ 2 levels "nao","sim": 2 2 2 2 2 2 2 2 2 2 ...
 $ CIGARRO  : Factor w/ 2 levels "nao","sim": 2 1 2 1 2 2 1 2 2 2 ...
 $ ALCOOL   : Factor w/ 2 levels "nao","sim": 1 2 1 2 2 2 2 1 2 1 ...
 $ GENERO   : Factor w/ 2 levels "F","M": 2 2 2 2 2 2 1 1 2 2 ...
 $ PRIMOG   : Factor w/ 2 levels "nao","sim": 2 1 1 1 1 2 2 1 2 1 ...
 $ PESO     : int  4592 4082 3175 4139 3679 3316 2948 3955 2778 2778 ...
 $ PREMAT   : int  0 -1 3 1 1 1 1 1 1 1 ...
 $ HOSP     : int  8 3 5 4 7 63 4 4 4 2 ...
 $ RENDA    : int  17000 9750 30000 27761 30000 39902 3072 16300 7996 2714 ...
 $ TRT      : Factor w/ 2 levels "caso","controle": 2 2 2 2 2 2 2 2 2 2 ...
 $ 

## Análise de Possíveis *Outliers*

Essa Análise se dará separando os tipos de tratamentos. Neste trabalho será considerado Outliers valores 1,5 * FIQ (Faixa Interquartil)  a baixo do primeiro quartil, ou acima do terceiro quartil.  

In [ ]:
# Exibe as informações estatísticas básica das variáveis
print("GRUPO DE TRATAMENTO Caso")
summary(CASO)
print("GRUPO DE TRATAMENTO Controle")
summary(CTRL)




[1] "GRUPO DE TRATAMENTO Caso"


     IDADE             ETNIA                      EDU       TRAB     PRENATAL 
 Min.   :13.00   branca   :117   medio              : 82   nao:119   nao: 13  
 1st Qu.:20.00   hispanica: 27   medio_incompleto   :126   sim:171   sim:277  
 Median :24.00   negra    :146   superior           : 34                      
 Mean   :24.44                   superior_incompleto: 48                      
 3rd Qu.:28.00                                                                
 Max.   :41.00                                                                
 CIGARRO   ALCOOL    GENERO  PRIMOG         PESO          PREMAT      
 nao:188   nao:254   F:143   nao:150   Min.   :1515   Min.   : 2.000  
 sim:102   sim: 36   M:147   sim:140   1st Qu.:1780   1st Qu.: 5.000  
                                       Median :2000   Median : 6.000  
                                       Mean   :2009   Mean   : 6.072  
                                       3rd Qu.:2240   3rd Qu.: 7.000  
                     

[1] "GRUPO DE TRATAMENTO Controle"


     IDADE             ETNIA                       EDU        TRAB     
 Min.   :16.00   branca   :2059   medio              :1738   nao:1551  
 1st Qu.:21.00   hispanica: 873   medio_incompleto   :1232   sim:2540  
 Median :24.00   negra    :1159   superior           : 332             
 Mean   :23.75                    superior_incompleto: 789             
 3rd Qu.:26.00                                                         
 Max.   :32.00                                                         
 PRENATAL   CIGARRO    ALCOOL     GENERO   PRIMOG          PESO     
 nao:  52   nao:2741   nao: 967   F:2064   nao:2368   Min.   :1502  
 sim:4039   sim:1350   sim:3124   M:2027   sim:1723   1st Qu.:2976  
                                                      Median :3345  
                                                      Mean   :3335  
                                                      3rd Qu.:3685  
                                                      Max.   :7597  
     PREMAT  

In [ ]:
#IDADE

print("GRUPO CASO")
## CASO
Q1 <- quantile(CASO$IDADE, probs = 0.25) #primeiro quartil
Q3 <- quantile(CASO$IDADE, probs = 0.75) # terceiro quartil
IQR <- Q3 - Q1 #indice
LimInf <- Q1 - 1.5*IQR #limite inferior
LimSup <- Q3 + 1.5*IQR #limite superior
OutlierIdade <- subset(CASO, IDADE < LimInf | IDADE > LimSup)
OutlierIdade

print("GRUPO CONTROLE")
## CONTROLE
Q1 <- quantile(CTRL$IDADE, probs = 0.25)
Q3 <- quantile(CTRL$IDADE, probs = 0.75)
IQR <- Q3 - Q1
LimInf <- Q1 - 1.5*IQR
LimSup <- Q3 + 1.5*IQR
OutlierIdade <- subset(CTRL, IDADE < LimInf | IDADE > LimSup)
OutlierIdade


[1] "GRUPO CASO"


IDADE ETNIA  EDU                 TRAB PRENATAL CIGARRO ALCOOL GENERO PRIMOG
265 41    branca superior_incompleto sim  sim      nao     nao    M      nao   
    PESO PREMAT HOSP RENDA TRT  QI PREMATCAT pesoCAT rendaCAT qiCAT
265 1600 7      16   71255 caso 95 alta      baixo   media    media

[1] "GRUPO CONTROLE"


IDADE ETNIA EDU TRAB PRENATAL CIGARRO ALCOOL GENERO PRIMOG PESO PREMAT
     HOSP RENDA TRT QI PREMATCAT pesoCAT rendaCAT qiCAT

Foi encontrado apenas um dado, para o grupo CASO, com um valor extremo por idade, contudo, essa observação vai de encontro com uma das hipóteses abordadas no trabalho, que mulheres com idades acima de 40 anos, tem uma maior possibilidade de ter um filho prematuro. Vale ressaltar que essa mãe nao fez uso de cigarro, alcool, e realizou o pré-natal.

In [ ]:
#PESO
print("GRUPO CASO")
## CASO
Q1 <- quantile(CASO$PESO, probs = 0.25)
Q3 <- quantile(CASO$PESO, probs = 0.75)
IQR <- Q3 - Q1
LimInf <- Q1 - 1.5*IQR
LimSup <- Q3 + 1.5*IQR
Outlier <- subset(CASO, PESO < LimInf | PESO > LimSup)
Outlier
print("GRUPO CONTROLE")
## CONTROLE
Q1 <- quantile(CTRL$PESO, probs = 0.25)
Q3 <- quantile(CTRL$PESO, probs = 0.75)
IQR <- Q3 - Q1
LimInfCTR <- Q1 - 1.5*IQR
LimSupCTR <- Q3 + 1.5*IQR
Outlier <- subset(CTRL, PESO < LimInfCTR | PESO > LimSupCTR)
cat("numero de observacoes de outliers para o Peso do Grupo Controle",
nrow(Outlier))

[1] "GRUPO CASO"


IDADE ETNIA EDU TRAB PRENATAL CIGARRO ALCOOL GENERO PRIMOG PESO PREMAT
     HOSP RENDA TRT QI PREMATCAT pesoCAT rendaCAT qiCAT

[1] "GRUPO CONTROLE"
numero de observacoes de outliers para o Peso do Grupo Controle 78

Na variável Peso, não foi obtido nenhum outlier para o grupo de tratamento do CASO, contudo no CONTROLE tiveram 78 observações. Abaixo, será analisado o motivo das 78 observações.

In [ ]:
#PESO

#abaixo de 2500 que e o limite para entrar no tratamento caso
print("Grupo Controle")
PesoBaixo <- subset(CTRL, PESO < 2500 & (PREMATCAT != "postermo" &
  PREMATCAT != "ausente"))
#abaixo ao limite inferior de outlier do grupo controle
PesoBaixoOutLier <- subset(CTRL, PESO < LimInfCTR & (PREMATCAT != "postermo" &
  PREMATCAT != "ausente"))
#abaixo ao limite inferior de outlier do grupo controle, considerando sem
#nenhuma prematuridade
PesoBaixoOutLierNoPremat <- subset(CTRL, PESO < LimInfCTR & (PREMATCAT
  == "postermo" | PREMATCAT == "ausente"))
cat("n° de observações para peso < 2500 e foram prematuros:" ,nrow(PesoBaixo))
cat('\n')
cat("n° de observações de outliers para peso < 2500 que foram prematuros:" ,
    nrow(PesoBaixoOutLier))
cat('\n')
print("Análise Estatística  de Peso para os Outliers de prematuridade postermo
  ou ausente para peso menor que 2500kg")
summary(PesoBaixoOutLierNoPremat)


[1] "Grupo Controle"
n° de observações para peso < 2500 e foram prematuros: 285
n° de observações de outliers para peso < 2500 que foram prematuros: 59
[1] "Análise Estatística  de Peso para os Outliers de prematuridade postermo\n  ou ausente para peso menor que 2500kg"


     IDADE          ETNIA                    EDU     TRAB   PRENATAL CIGARRO
 Min.   :23   branca   :0   medio              :0   nao:1   nao:0    nao:0  
 1st Qu.:23   hispanica:0   medio_incompleto   :1   sim:0   sim:1    sim:1  
 Median :23   negra    :1   superior           :0                           
 Mean   :23                 superior_incompleto:0                           
 3rd Qu.:23                                                                 
 Max.   :23                                                                 
 ALCOOL  GENERO PRIMOG       PESO          PREMAT        HOSP       RENDA     
 nao:0   F:0    nao:1   Min.   :1842   Min.   :-2   Min.   :2   Min.   :4440  
 sim:1   M:1    sim:0   1st Qu.:1842   1st Qu.:-2   1st Qu.:2   1st Qu.:4440  
                        Median :1842   Median :-2   Median :2   Median :4440  
                        Mean   :1842   Mean   :-2   Mean   :2   Mean   :4440  
                        3rd Qu.:1842   3rd Qu.:-2   3rd Qu.:2   3r

In [ ]:
PesoBaixo <- subset(CTRL, PESO < 2500 & (PREMATCAT != "postermo" &
  PREMATCAT != "ausente"))
summary  (PesoBaixo)

     IDADE             ETNIA                      EDU       TRAB     PRENATAL 
 Min.   :17.00   branca   :116   medio              :124   nao:134   nao:  7  
 1st Qu.:21.00   hispanica: 63   medio_incompleto   :104   sim:151   sim:278  
 Median :23.00   negra    :106   superior           : 13                      
 Mean   :23.22                   superior_incompleto: 44                      
 3rd Qu.:26.00                                                                
 Max.   :31.00                                                                
 CIGARRO   ALCOOL    GENERO  PRIMOG         PESO          PREMAT      
 nao:177   nao: 60   F:130   nao:163   Min.   :1502   Min.   : 1.000  
 sim:108   sim:225   M:155   sim:122   1st Qu.:1984   1st Qu.: 1.000  
                                       Median :2268   Median : 4.000  
                                       Mean   :2187   Mean   : 4.235  
                                       3rd Qu.:2409   3rd Qu.: 6.000  
                     

Temos 285 observações no tratamento controle, que se encaixam no tratamento CASO, sendo 59 observações que ficaram como outlier de Peso no grupo contole. Observamos também um RCN póstermo, no qual a mãe era jovem de etnia negra, e fazia uso de cigarro e alcool, o seu RCN passou duas semanas a mais para nascer, e passou duas semanas hospitalizados, apresentando um QI baixo, reforçando também a nossa hipótese de que o cigarro e o alcool podem afetar a prematuridade e o desenvolvimento cognitivo.


In [ ]:
#PESO
#acima do limite superior de outlier do grupo controle
PesoAcimaOutLier <- subset(CTRL, PESO > LimSupCTR)
print("analise estastiticas para peso com valores altos extremos")
summary(PesoAcimaOutLier)
#foi verificado 18 outliers, sendo 9 para postermo, 1 ausente, e 8
#com prematuridade
PesoAcimaOutLierNoPremat <- subset(CTRL, PESO > LimSupCTR & (PREMATCAT
  == "postermo" | PREMATCAT == "ausente"))
print("analise estastiticas para peso com valores altos extremos sendo
uma prematuridade posterma ou ausente ")
summary(PesoAcimaOutLierNoPremat)
print("dataframe dos valores para peso extremo para sem prematuridade,
ou posterma")
head(PesoAcimaOutLierNoPremat, 10)

[1] "analise estastiticas para peso com valores altos extremos"


     IDADE             ETNIA                    EDU     TRAB    PRENATAL
 Min.   :18.00   branca   :9   medio              :5   nao:13   nao: 0  
 1st Qu.:22.00   hispanica:1   medio_incompleto   :9   sim: 5   sim:18  
 Median :23.50   negra    :8   superior           :1                    
 Mean   :23.83                 superior_incompleto:3                    
 3rd Qu.:26.00                                                          
 Max.   :30.00                                                          
 CIGARRO  ALCOOL   GENERO PRIMOG        PESO          PREMAT       
 nao:12   nao: 5   F:12   nao:16   Min.   :4762   Min.   :-7.0000  
 sim: 6   sim:13   M: 6   sim: 2   1st Qu.:4819   1st Qu.:-2.0000  
                                   Median :4946   Median :-0.5000  
                                   Mean   :5315   Mean   : 0.3333  
                                   3rd Qu.:5329   3rd Qu.: 2.5000  
                                   Max.   :7597   Max.   :14.0000  
      HOSP   

[1] "analise estastiticas para peso com valores altos extremos sendo\numa prematuridade posterma ou ausente "


     IDADE             ETNIA                    EDU     TRAB   PRENATAL CIGARRO
 Min.   :19.00   branca   :6   medio              :3   nao:7   nao: 0   nao:6  
 1st Qu.:21.25   hispanica:0   medio_incompleto   :4   sim:3   sim:10   sim:4  
 Median :22.50   negra    :4   superior           :0                           
 Mean   :22.90                 superior_incompleto:3                           
 3rd Qu.:24.75                                                                 
 Max.   :27.00                                                                 
 ALCOOL  GENERO PRIMOG       PESO          PREMAT          HOSP      
 nao:2   F:8    nao:9   Min.   :4762   Min.   :-7.0   Min.   : 1.00  
 sim:8   M:2    sim:1   1st Qu.:4890   1st Qu.:-3.0   1st Qu.: 3.00  
                        Median :4989   Median :-2.0   Median : 3.00  
                        Mean   :5448   Mean   :-2.3   Mean   : 9.60  
                        3rd Qu.:5329   3rd Qu.:-1.0   3rd Qu.:12.25  
                    

[1] "dataframe dos valores para peso extremo para sem prematuridade,\nou posterma"


IDADE ETNIA  EDU                 TRAB PRENATAL CIGARRO ALCOOL GENERO
572  25    branca medio_incompleto    nao  sim      sim     sim    M     
650  27    branca superior_incompleto nao  sim      sim     sim    F     
1778 23    negra  superior_incompleto nao  sim      sim     sim    M     
1931 22    branca medio_incompleto    nao  sim      nao     sim    F     
1961 26    branca medio_incompleto    nao  sim      sim     sim    F     
2205 21    branca medio               sim  sim      nao     nao    F     
2457 20    negra  medio               nao  sim      nao     sim    F     
2664 19    negra  medio_incompleto    nao  sim      nao     sim    F     
2672 24    negra  superior_incompleto sim  sim      nao     nao    F     
3274 22    branca medio               sim  sim      nao     sim    F     
     PRIMOG PESO PREMAT HOSP RENDA  TRT      QI  PREMATCAT pesoCAT rendaCAT
572  nao    5414 -2      3     8071 controle  93 postermo  normal  baixa   
650  nao    7030 -1      2    36500 controle  90 postermo  normal  baixa   
1778 nao    4762 -3      1    56600 controle  89 postermo  normal  media   
1931 nao    4989 -1      7    39640 controle  82 postermo  normal  baixa   
1961 nao    7597  0      3     9138 controle  95 ausente   normal  baixa   
2205 nao    4876 -3      3    16050 controle  97 postermo  normal  baixa   
2457 sim    4932 -2     30    34828 controle  61 postermo  normal  baixa   
2664 nao    5074 -3     30    36514 controle  33 postermo  normal  baixa   
2672 nao    4989 -7     14    23000 controle  88 postermo  normal  baixa   
3274 nao    4819 -1      3   -13200 controle 102 postermo  normal  baixa   
     qiCAT
572  media
650  baixa
1778 baixa
1931 baixa
1961 media
2205 media
2457 baixa
2664 baixa
2672 baixa
3274 media

Ao analisar os possíveis outiliers de peso, vimos que dois rcn apresentaram uma
hospitalizacao de 30 semanas, e um outro de 14 semanas, podendo ser alguma doença relacionado ao parto. E todos os bebês que tiveram um peso elevado apresentaram um qi baixo ou médio.

In [ ]:
#PESO
# verificacao dos 8 dados de outlier com peso alto para prematuridade

PesoAcimaOutLierComPremat <- subset(CTRL, PESO > LimSupCTR & (PREMATCAT
  == "moderada" | PREMATCAT == "alta" | PREMATCAT == "extrema"))
print("ANALISE ESTASTITICA PARA PESO COM LIMITES SUPERIORES ALTOS COM ALGUM
NIVEL DE PREMATURIDADE")
summary(PesoAcimaOutLierComPremat)
head(PesoAcimaOutLierComPremat, 10)

[1] "ANALISE ESTASTITICA PARA PESO COM LIMITES SUPERIORES ALTOS COM ALGUM\nNIVEL DE PREMATURIDADE"


     IDADE            ETNIA                    EDU     TRAB   PRENATAL CIGARRO
 Min.   :18.0   branca   :3   medio              :2   nao:6   nao:0    nao:6  
 1st Qu.:22.0   hispanica:1   medio_incompleto   :5   sim:2   sim:8    sim:2  
 Median :26.0   negra    :4   superior           :1                           
 Mean   :25.0                 superior_incompleto:0                           
 3rd Qu.:27.5                                                                 
 Max.   :30.0                                                                 
 ALCOOL  GENERO PRIMOG       PESO          PREMAT            HOSP     
 nao:3   F:4    nao:7   Min.   :4762   Min.   : 1.000   Min.   : 2.0  
 sim:5   M:4    sim:1   1st Qu.:4812   1st Qu.: 1.000   1st Qu.: 3.0  
                        Median :4819   Median : 3.000   Median : 6.0  
                        Mean   :5149   Mean   : 3.625   Mean   :15.5  
                        3rd Qu.:5152   3rd Qu.: 3.000   3rd Qu.: 7.0  
                     

IDADE ETNIA     EDU              TRAB PRENATAL CIGARRO ALCOOL GENERO
546  30    branca    medio            sim  sim      nao     sim    M     
1618 29    branca    medio_incompleto nao  sim      sim     nao    F     
1954 26    branca    medio            nao  sim      nao     nao    M     
2665 22    negra     medio_incompleto nao  sim      nao     sim    F     
3294 27    negra     superior         sim  sim      nao     sim    F     
3436 26    negra     medio_incompleto nao  sim      nao     sim    M     
3474 18    negra     medio_incompleto nao  sim      sim     sim    F     
3732 22    hispanica medio_incompleto nao  sim      nao     nao    M     
     PRIMOG PESO PREMAT HOSP RENDA TRT      QI PREMATCAT pesoCAT rendaCAT qiCAT
546  sim    5726 14     90   13436 controle 94 extrema   normal  baixa    media
1618 nao    4819  1      3   11832 controle 82 moderada  normal  baixa    baixa
1954 nao    4819  3      6   15440 controle 85 moderada  normal  baixa    baixa
2665 nao    4762  1      7    1452 controle 91 moderada  normal  baixa    media
3294 nao    4819  3      7   52160 controle 85 moderada  normal  media    baixa
3436 nao    4961  1      6   40000 controle 76 moderada  normal  baixa    baixa
3474 nao    6492  3      2    2880 controle 66 moderada  normal  baixa    baixa
3732 nao    4791  3      3   20000 controle 80 moderada  normal  baixa    baixa

Podemos observar entre os possíveis outliers, um RCN, que tem uma prematuridade de 14 semanas (extrema) e passou 90 semanas hospitalizado. Ou pode ser um erro de digitação ou alguma doença apresentada na hora do nascimento.

In [ ]:
#PREMATURIDADE
## CASO
Q1 <- quantile(CASO$PREMAT, probs = 0.25)
Q3 <- quantile(CASO$PREMAT, probs = 0.75)
IQR <- Q3 - Q1
LimInf <- Q1 - 1.5*IQR
LimSup <- Q3 + 1.5*IQR
Outlier <- subset(CASO, PREMAT < LimInf | PREMAT > LimSup)
cat("Analise de Outlier de prematuridade para o tipo caso")
Outlier


## CONTROLE
## PREMATURIDADE POSTERMO
cat("Analise de Outlier de prematuridade para o tipo controle")
cat('\n')
print("Considerando apenas postermo")
POStermo <- subset (CTRL, PREMAT < 0)
#SERAO ANALISADOS VALORES DE POSTERMOS MAIORES IGUAL A - 4
#DEVIDO AO LIMITE INFERIOR SER 3.5
print("Foi gerado os dados em que o postermo é superior a 4, esses valores
podem ser considerados extremos")
Outlier <- subset(POStermo, PREMAT <= -4)
head(Outlier, 25)

## PREMATURIDADE ausente
cat("Considerando ausencia de prematuridade NO GRUPO CONTROLE")
PrematuridadeAus <- subset (CTRL, PREMAT == 0)
summary(PrematuridadeAus)

#PREMATURIDADE ALTA E EXTREMA
cat("Considerando apenas prematuridade alta e extrema NO GRUPO CONTROLE")
Prematuridade <- subset(CTRL, PREMAT > 6)
Q1 <- quantile(Prematuridade$PREMAT, probs = 0.25)
Q3 <- quantile(Prematuridade$PREMAT, probs = 0.75)
IQR <- Q3 - Q1
LimInf <- Q1 - 1.5*IQR
LimSup <- Q3 + 1.5*IQR
print(paste("O valor para limite superior considerado foi ", LimSup))
cat('\n')
print("analisando apenas os dados com o valor de limite superior,
pois, o objetivo sao os valores extremos da variavel em analise")
Outlier <- subset(Prematuridade,PREMAT > LimSup)
summary(Outlier)
print("dados com valores extremos de prematuridade ")
head(Outlier)

Analise de Outlier de prematuridade para o tipo caso

IDADE ETNIA  EDU                 TRAB PRENATAL CIGARRO ALCOOL GENERO PRIMOG
179 23    negra  superior_incompleto sim  sim      nao     nao    F      sim   
186 20    branca medio_incompleto    nao  sim      sim     sim    M      sim   
252 32    branca superior_incompleto sim  sim      nao     nao    M      sim   
    PESO PREMAT HOSP RENDA TRT  QI PREMATCAT pesoCAT rendaCAT qiCAT
179 1540 11     26    1531 caso 92 extrema   baixo   baixa    media
186 1550 11     51   12500 caso 72 extrema   baixo   baixa    baixa
252 1780 11     40   42500 caso 87 extrema   baixo   baixa    baixa

Analise de Outlier de prematuridade para o tipo controle
[1] "Considerando apenas postermo"
[1] "Foi gerado os dados em que o postermo é superior a 4, esses valores\npodem ser considerados extremos"


IDADE ETNIA     EDU                 TRAB PRENATAL CIGARRO ALCOOL GENERO
628  21    branca    medio_incompleto    sim  sim      sim     sim    F     
685  20    negra     medio               sim  sim      sim     sim    F     
794  22    hispanica medio_incompleto    sim  sim      nao     sim    F     
865  23    branca    medio               sim  sim      sim     nao    M     
930  21    hispanica medio_incompleto    nao  sim      nao     sim    F     
1037 19    branca    medio               sim  sim      nao     nao    M     
1164 28    branca    medio               sim  sim      sim     sim    M     
1289 19    branca    medio               nao  sim      nao     sim    F     
1364 25    branca    medio               sim  sim      sim     nao    F     
1580 19    branca    medio               nao  sim      sim     nao    M     
1591 21    branca    medio_incompleto    nao  sim      sim     sim    M     
1685 23    negra     medio_incompleto    nao  sim      sim     sim    F     
1795 20    branca    medio               sim  sim      sim     nao    M     
1808 21    negra     superior_incompleto sim  sim      nao     nao    M     
2263 23    branca    medio_incompleto    sim  sim      sim     sim    F     
2317 23    branca    medio_incompleto    sim  sim      sim     nao    F     
2359 23    hispanica medio_incompleto    sim  sim      nao     sim    F     
2462 23    hispanica medio               sim  sim      nao     sim    M     
2672 24    negra     superior_incompleto sim  sim      nao     nao    F     
2736 20    branca    medio_incompleto    sim  sim      nao     sim    M     
2793 24    hispanica superior            sim  sim      nao     sim    M     
3179 23    branca    medio               sim  sim      nao     sim    M     
4009 27    hispanica medio_incompleto    sim  sim      sim     nao    M     
4182 27    hispanica superior_incompleto sim  sim      nao     sim    F     
4344 20    branca    medio               sim  sim      sim     sim    M     
     PRIMOG PESO PREMAT HOSP RENDA  TRT      QI  PREMATCAT pesoCAT rendaCAT
628  nao    3345 -5      3    14095 controle  69 postermo  normal  baixa   
685  sim    3572 -4      7     2960 controle  81 postermo  normal  baixa   
794  nao    4706 -4      0    10368 controle  60 postermo  normal  baixa   
865  nao    3175 -4      3     2583 controle  92 postermo  normal  baixa   
930  sim    4422 -7      3    13370 controle  78 postermo  normal  baixa   
1037 sim    3912 -4      6    16550 controle  95 postermo  normal  baixa   
1164 nao    4541 -5      1    44700 controle  71 postermo  normal  baixa   
1289 sim    4195 -4      5     2000 controle  92 postermo  normal  baixa   
1364 sim    4054 -6      7    45272 controle 102 postermo  normal  baixa   
1580 sim    3515 -5      3    16520 controle 102 postermo  normal  baixa   
1591 nao    2749 -5      4     5904 controle  67 postermo  normal  baixa   
1685 nao    3402 -5      7    33998 controle  69 postermo  normal  baixa   
1795 sim    3713 -4      1   -36577 controle  90 postermo  normal  baixa   
1808 nao    4139 -7      7     3495 controle  78 postermo  normal  baixa   
2263 nao    3316 -4      4     7402 controle  68 postermo  normal  baixa   
2317 sim    3624 -4      0      400 controle  88 postermo  normal  baixa   
2359 nao    3061 -4      1     5090 controle  44 postermo  normal  baixa   
2462 nao    3316 -4      3    11430 controle  89 postermo  normal  baixa   
2672 nao    4989 -7     14    23000 controle  88 postermo  normal  baixa   
2736 nao    3628 -5      3    18392 controle 104 postermo  normal  baixa   
2793 sim    3118 -4      4    46903 controle  77 postermo  normal  baixa   
3179 sim    4082 -5      5    34100 controle 111 postermo  normal  baixa   
4009 nao    4167 -4      2     2122 controle 118 postermo  normal  baixa   
4182 sim    4025 -5      3    12101 controle  82 postermo  normal  baixa   
4344 sim    3572 -4      7    17000 controle  84 postermo  normal  baixa   
     qiCAT
628  baixa
685  

Considerando ausencia de prematuridade NO GRUPO CONTROLE

     IDADE            ETNIA                     EDU      TRAB     PRENATAL 
 Min.   :17.0   branca   :84   medio              :71   nao:100   nao:  2  
 1st Qu.:21.0   hispanica:45   medio_incompleto   :79   sim:100   sim:198  
 Median :23.0   negra    :71   superior           :15                      
 Mean   :23.4                  superior_incompleto:35                      
 3rd Qu.:26.0                                                              
 Max.   :31.0                                                              
 CIGARRO   ALCOOL    GENERO  PRIMOG         PESO          PREMAT 
 nao:137   nao: 54   F:100   nao:117   Min.   :1932   Min.   :0  
 sim: 63   sim:146   M:100   sim: 83   1st Qu.:3196   1st Qu.:0  
                                       Median :3458   Median :0  
                                       Mean   :3506   Mean   :0  
                                       3rd Qu.:3742   3rd Qu.:0  
                                       Max.   :7597   Max.   :0  
      

Considerando apenas prematuridade alta e extrema NO GRUPO CONTROLE[1] "O valor para limite superior considerado foi  12"

[1] "analisando apenas os dados com o valor de limite superior,\npois, o objetivo sao os valores extremos da variavel em analise"


     IDADE             ETNIA                    EDU     TRAB   PRENATAL CIGARRO
 Min.   :19.00   branca   :1   medio              :2   nao:2   nao:1    nao:4  
 1st Qu.:22.75   hispanica:1   medio_incompleto   :1   sim:2   sim:3    sim:0  
 Median :26.00   negra    :2   superior           :0                           
 Mean   :25.25                 superior_incompleto:1                           
 3rd Qu.:28.50                                                                 
 Max.   :30.00                                                                 
 ALCOOL  GENERO PRIMOG       PESO          PREMAT           HOSP       
 nao:0   F:3    nao:3   Min.   :1814   Min.   :13.00   Min.   :  7.00  
 sim:4   M:1    sim:1   1st Qu.:1814   1st Qu.:13.00   1st Qu.:  7.75  
                        Median :1848   Median :13.00   Median : 49.00  
                        Mean   :2809   Mean   :13.25   Mean   : 51.25  
                        3rd Qu.:2844   3rd Qu.:13.25   3rd Qu.: 92.50  
        

[1] "dados com valores extremos de prematuridade "


IDADE ETNIA     EDU                 TRAB PRENATAL CIGARRO ALCOOL GENERO
546  30    branca    medio               sim  sim      nao     sim    M     
951  24    hispanica medio_incompleto    sim  nao      nao     sim    F     
1150 28    negra     superior_incompleto nao  sim      nao     sim    F     
2616 19    negra     medio               nao  sim      nao     sim    F     
     PRIMOG PESO PREMAT HOSP RENDA TRT      QI PREMATCAT pesoCAT rendaCAT qiCAT
546  sim    5726 14      90  13436 controle 94 extrema   normal  baixa    media
951  nao    1883 13     100  10800 controle 38 extrema   baixo   baixa    baixa
1150 nao    1814 13       8  16800 controle 74 extrema   baixo   baixa    baixa
2616 nao    1814 13       7  12000 controle 80 extrema   baixo   baixa    baixa

Podemos observar na prematuridade Póstermo, alguns outliers relacionados a QI, onde um RCN postermo apresentou um QI muito abaixo (44), e uma prematuridade postermo muito elevada com 7 semanas, e passando 14 semanas hospitalizado.

Já no Grupo de tratamento do tipo caso, os 3 valores extremos de prematuridade, apresentaram um alto indice de hospitalização. Contudo todos esses valores sao factíveis para acontecer.

Na Prematuridade ausente, já foi analisado no outlier do Peso, o caso em que o PESO APRESENTOU 7KG, foi apresentado também alguns QI muito baixos, e algumas hospitalizacoes elevadas.

Na Prematuridade alta e extrema, foram analisados os valores extremos, o limite superior foi valores acima de 12 semanas de prematuridade, ao realizar a filtragem, foram observados 4 dados, pode-se observar dois valores altos de hospitalização podendo sinalizar que os RCN ficaram quase DOIS ANOS, em hospitalizacao, esse caso mais extremo, apresentou um RCN com um QI muito abaixo, vale ressaltar, que esse caso era para ter recebido tratamento do Governo, pois o peso foi abaixo de 2500 kg.

## Análise da variável prematuridade

In [ ]:
# Exibe as informações estatísticas básica das variáveis
summary(IHDP)

     IDADE            ETNIA                       EDU        TRAB     
 Min.   :13.0   branca   :2176   medio              :1820   nao:1670  
 1st Qu.:21.0   hispanica: 900   medio_incompleto   :1358   sim:2711  
 Median :24.0   negra    :1305   superior           : 366             
 Mean   :23.8                    superior_incompleto: 837             
 3rd Qu.:26.0                                                         
 Max.   :41.0                                                         
 PRENATAL   CIGARRO    ALCOOL     GENERO   PRIMOG          PESO     
 nao:  65   nao:2929   nao:1221   F:2207   nao:2518   Min.   :1502  
 sim:4316   sim:1452   sim:3160   M:2174   sim:1863   1st Qu.:2891  
                                                      Median :3288  
                                                      Mean   :3247  
                                                      3rd Qu.:3657  
                                                      Max.   :7597  
     PREMAT         

In [ ]:
# Exibe as informações estatísticas básica da variável 'PREMATURIDADE'
summary(IHDP$PREMATCAT)
summary(IHDP$PREMAT)

postermo  ausente moderada     alta  extrema 
     707      200     3165      242       67

   Min. 1st Qu.  Median    Mean 3rd Qu.    Max. 
 -7.000   1.000   1.000   1.493   2.000  14.000 

Ao analisar a variável prematuridade por categoria, temos a quantidade por cada caso da estratificação. Ao analisar os valores absolutos, podemos ver que temos dois valores extremos, o max e o minimo, entrando nos casos de nascimento tardio para o valor mínimo e uma prematuridade extrema para o valor do máximo. Ainda podemos observar que a média baixa (1.4 semanas) reflete a grande quantidade de dados para as observações moderadas ( 1 a 6 semanas).

In [ ]:
# Tabela de frequência absoluta de mães por 'PREMATURIDADE' e por 'TRT' do filho(a)
tabPT <- table(IHDP$PREMATCAT, IHDP$TRT)
tabPT

          
           caso controle
  postermo    0      707
  ausente     0      200
  moderada  104     3061
  alta      153       89
  extrema    33       34

###1. Mulheres Jovens (A baixo de 20 anos), principalmenete de etnia negra (devido ao contexto do Estado norte americano) teriam maiores chances de apresentarem uma prematuridade.

O Grupo caso não apresenta nenhum valor para o nascimento tardio, ou com uma prematuridade ausente. Isso faz sentido, pois o programa visa trabalhar com bebês prematuros e abaixo de 2500 kg.

In [ ]:
# Tabela da frequência relativa avaliando mulheres com idade menor ou igual a 20 e etnia

CASO20 <- CASO[(CASO[,'IDADE'] <= 20),]
CTRL20 <- CTRL[(CTRL[,'IDADE'] <= 20),]

# Frequência calculadas nas linhas
cat('Grupo CASO')
tablecaso <- table(CASO20$PREMATCAT, CASO20$ETNIA)
tablecaso1 <- prop.table(tablecaso, margin = 1)
round(tablecaso1*100,2)
cat('\n')
cat('Grupo CTRL')
tablectrl <- table(CTRL20$PREMATCAT, CTRL20$ETNIA)
tablectrl1 <- prop.table(tablectrl, margin = 1)
round(tablectrl1*100,2)

Grupo CASO

          
           branca hispanica negra
  postermo                       
  ausente                        
  moderada  17.86      7.14 75.00
  alta      33.33     17.65 49.02
  extrema   11.11     11.11 77.78


Grupo CTRL

          
           branca hispanica negra
  postermo  50.00     22.39 27.61
  ausente   55.26     13.16 31.58
  moderada  39.62     22.64 37.74
  alta      77.78      5.56 16.67
  extrema   30.00     20.00 50.00

Observando a prematuridade em mulheres com menos que 20 anos no grupo caso, verifica-se que a maioria dos casos nas 3 categorias (moderada, alta e extrema) acontecendo com mulheres negras. Já no grupo controle, os casos de prematuridade moderada e alta concentram-se em mulheres da etnia branca. Com o grupo controle, observe que realmente a maior frequência nos casos de prematuridade extrema é em mulheres negras.

In [ ]:
# Frequência calculadas nas colunas
cat('Grupo CASO')
tablecaso <- table(CASO20$PREMATCAT, CASO20$ETNIA)
tablecaso1 <- prop.table(tablecaso, margin = 2)
round(tablecaso1*100,2)
cat('\n')
cat('Grupo CTRL')
tablectrl <- table(CTRL20$PREMATCAT, CTRL20$ETNIA)
tablectrl1 <- prop.table(tablectrl, margin = 2)
round(tablectrl1*100,2)

Grupo CASO

          
           branca hispanica negra
  postermo   0.00      0.00  0.00
  ausente    0.00      0.00  0.00
  moderada  21.74     16.67 39.62
  alta      73.91     75.00 47.17
  extrema    4.35      8.33 13.21


Grupo CTRL

          
           branca hispanica negra
  postermo  22.79     20.55 15.61
  ausente    7.14      3.42  5.06
  moderada  64.29     73.97 75.95
  alta       4.76      0.68  1.27
  extrema    1.02      1.37  2.11

Observando agora as frequências com relação as colunas da tabela (etnia), verifica-se que no grupo caso a maioria dos ocorrências em mulheres brancas, hispânicas e negras com menos de 20 anos são de prematuridade alta. Já no grupo controle a maioria nas 3 etnias são de prematurudade moderada.

### 2. Mulheres que tenham uma idade maior que 20 anos que são de etnias negras apresentem uma maior prematuridade.

In [ ]:
# Tabela da frequência relativa avaliando mulheres com idade menor ou igual a 20 e etnia

CASO20 <- CASO[(CASO[,'IDADE'] > 20),]
CTRL20 <- CTRL[(CTRL[,'IDADE'] > 20),]

# Frequência calculadas nas linhas
cat('Grupo CASO')
tablecaso <- table(CASO20$PREMATCAT, CASO20$ETNIA)
tablecaso1 <- prop.table(tablecaso, margin = 1)
round(tablecaso1*100,2)
cat('\n')
cat('Grupo CTRL')
tablectrl <- table(CTRL20$PREMATCAT, CTRL20$ETNIA)
tablectrl1 <- prop.table(tablectrl, margin = 1)
round(tablectrl1*100,2)

Grupo CASO

          
           branca hispanica negra
  postermo                       
  ausente                        
  moderada  42.11      6.58 51.32
  alta      50.98      7.84 41.18
  extrema   41.67      8.33 50.00


Grupo CTRL

          
           branca hispanica negra
  postermo  57.94     17.98 24.08
  ausente   38.89     24.69 36.42
  moderada  50.85     21.67 27.48
  alta      64.79     22.54 12.68
  extrema   41.67     33.33 25.00

Como pode ser observado, no grupo caso, boa parte das mulheres que tenham acima de 20 anos ou sao negras, ou sao brancas, tendo um bom numero quando o parto é prematuro extremo, já no controle, é observado uma maior frequência de etnias branca, em todo o estrato.

In [ ]:
# Frequência calculadas nas colunas
cat('Grupo CASO')
tablecaso <- table(CASO20$PREMATCAT, CASO20$ETNIA)
tablecaso1 <- prop.table(tablecaso, margin = 2)
round(tablecaso1*100,2)
cat('\n')
cat('Grupo CTRL')
tablectrl <- table(CTRL20$PREMATCAT, CTRL20$ETNIA)
tablectrl1 <- prop.table(tablectrl, margin = 2)
round(tablectrl1*100,2)

Grupo CASO

          
           branca hispanica negra
  postermo   0.00      0.00  0.00
  ausente    0.00      0.00  0.00
  moderada  34.04     33.33 41.94
  alta      55.32     53.33 45.16
  extrema   10.64     13.33 12.90


Grupo CTRL

          
           branca hispanica negra
  postermo  18.81     14.17 14.97
  ausente    3.57      5.50  6.40
  moderada  74.45     77.03 77.01
  alta       2.61      2.20  0.98
  extrema    0.57      1.10  0.65

Analisando a frequência por coluna, pode ser notado que a maior frequência de nascimentos prematuros de mulheres adultas negras ou hispanicas no grupo caso sao alto, ou moderado, já no controle, para quaisquer etnia a maior concentração de observações é no prematuro moderado.

### 3. Efeito do Cigarro na Prematuridade, como também avaliando as consequências no desenvolvimento intelectual da criança.

In [ ]:
# Tabela da frequência relativa avaliando mulheres que usaram o cigarro e QI do filho

CASO20 <- CASO[(CASO[,'CIGARRO'] == 'sim'),]
CTRL20 <- CTRL[(CTRL[,'CIGARRO'] == 'sim'),]

# Frequência calculadas nas linhas
cat('Grupo CASO')
tablecaso <- table(CASO20$PREMATCAT, CASO20$qiCAT)
tablecaso1 <- prop.table(tablecaso, margin = 1)
round(tablecaso1*100,2)
cat('\n')
cat('Grupo CTRL')
tablectrl <- table(CTRL20$PREMATCAT, CTRL20$qiCAT)
tablectrl1 <- prop.table(tablectrl, margin = 1)
round(tablectrl1*100,2)


Grupo CASO

          
           baixa media  alta
  postermo                  
  ausente                   
  moderada 51.22 39.02  9.76
  alta     38.89 40.74 20.37
  extrema  28.57 42.86 28.57


Grupo CTRL

          
           baixa media  alta
  postermo 50.00 41.11  8.89
  ausente  60.32 36.51  3.17
  moderada 56.26 35.52  8.21
  alta     33.33 54.55 12.12
  extrema  90.00 10.00  0.00

No grupo caso, mulheres que fumaram e tiveram prematuridade moderada, o bebe ficou com QI baixo. Já mulheres que fumaram e tiveram prematuridade alta e extrema, a maioria dos bebes ficaram com QI médio. No grupo controle essa observção muda na prematuridade extrema em que a maioria dos bebes ficaram com QI baixo.

In [ ]:
# Frequência calculadas nas colunas
cat('Grupo CASO')
tablecaso <- table(CASO20$PREMATCAT, CASO20$qiCAT)
tablecaso1 <- prop.table(tablecaso, margin = 2)
round(tablecaso1*100,2)
cat('\n')
cat('Grupo CTRL')
tablectrl <- table(CTRL20$PREMATCAT, CTRL20$qiCAT)
tablectrl1 <- prop.table(tablectrl, margin = 2)
round(tablectrl1*100,2)

Grupo CASO

          
           baixa media  alta
  postermo  0.00  0.00  0.00
  ausente   0.00  0.00  0.00
  moderada 47.73 39.02 23.53
  alta     47.73 53.66 64.71
  extrema   4.55  7.32 11.76


Grupo CTRL

          
           baixa media  alta
  postermo 18.22 22.24 21.82
  ausente   5.13  4.61  1.82
  moderada 73.95 69.34 72.73
  alta      1.48  3.61  3.64
  extrema   1.21  0.20  0.00

A observação por coluna segue a mesma lógica vista nas outras estratificações. Grupo caso concentrando observações na prematuridade alta enquanto o grupo controle a maioria das observações estão em prematuridade moderada.

###4. Efeito do Álcool na prematuridade, como também avaliando as suas consequências no desenvolvimento intelectual da criança.

In [ ]:
# Tabela da frequência relativa avaliando mulheres que usaram
#Alcool e QI do filho

CASO20 <- CASO[(CASO[,'ALCOOL'] == 'sim'),]
CTRL20 <- CTRL[(CTRL[,'ALCOOL'] == 'sim'),]

# Frequência calculadas nas linhas
cat('Grupo CASO')
tablecaso <- table(CASO20$PREMATCAT, CASO20$qiCAT)
tablecaso1 <- prop.table(tablecaso, margin = 1)
round(tablecaso1*100,2)
cat('\n')
cat('Grupo CTRL')
tablectrl <- table(CTRL20$PREMATCAT, CTRL20$qiCAT)
tablectrl1 <- prop.table(tablectrl, margin = 1)
round(tablectrl1*100,2)

Grupo CASO

          
           baixa media  alta
  postermo                  
  ausente                   
  moderada 44.44 38.89 16.67
  alta     26.67 46.67 26.67
  extrema  66.67 33.33  0.00


Grupo CTRL

          
           baixa media  alta
  postermo 55.83 33.83 10.34
  ausente  60.96 31.51  7.53
  moderada 57.09 34.06  8.85
  alta     43.94 42.42 13.64
  extrema  77.42 19.35  3.23

Ao analisar o frequencia por linha das mães que fizeram uso de alcool durante a gravidez, observamos que no grupo caso, as mães que tiveram uma prematuridade extrema e fizeram o uso de álcool, os RCN´s, tiveram ou um QI Baixo, ou um QI médio, indo de encontro com a nossa hipótese levantada, principalmente quando se analisa o grupo controle, onde a utilização do álcool, juntamente com uma prematuridade, seja de qualquer nível impactou diretamente no QI da criança.

In [ ]:
# Tabela da frequência relativa avaliando mulheres que usaram
#Alcool e QI do filho

CASO20 <- CASO[(CASO[,'ALCOOL'] == 'sim'),]
CTRL20 <- CTRL[(CTRL[,'ALCOOL'] == 'sim'),]

# Frequência calculadas nas colunas
cat('Grupo CASO')
tablecaso <- table(CASO20$PREMATCAT, CASO20$qiCAT)
tablecaso1 <- prop.table(tablecaso, margin = 2)
round(tablecaso1*100,2)
cat('\n')
cat('Grupo CTRL')
tablectrl <- table(CTRL20$PREMATCAT, CTRL20$qiCAT)
tablectrl1 <- prop.table(tablectrl, margin = 2)
round(tablectrl1*100,2)

Grupo CASO

          
           baixa media  alta
  postermo  0.00  0.00  0.00
  ausente   0.00  0.00  0.00
  moderada 57.14 46.67 42.86
  alta     28.57 46.67 57.14
  extrema  14.29  6.67  0.00


Grupo CTRL

          
           baixa media  alta
  postermo 16.69 16.98 19.37
  ausente   5.00  4.34  3.87
  moderada 75.34 75.47 73.24
  alta      1.63  2.64  3.17
  extrema   1.35  0.57  0.35

A observação por coluna no Grupo CASO, percebe-se uma maior concentração em uma prematuridade moderada e alta, principalmente quando se trata de um QI médio. No grupo controle, as observações seguem o mesmo padrão das anteriores se concentrando mais em uma prematuridade moderada.

### 5. Mulheres abaixo de 20 anos e que não fizeram um prenatal apresentem maiores chances de terem bebês prematuros.

In [ ]:
# Tabela da frequência relativa avaliando mulheres com idade menor ou igual a 20 e prenatal

CASO20 <- CASO[(CASO[,'IDADE'] <= 20),]
CTRL20 <- CTRL[(CTRL[,'IDADE'] <= 20),]

cat('Grupo CASO')
tablecaso <- table(CASO20$PREMATCAT, CASO20$PRENATAL)
tablecaso1 <- prop.table(tablecaso, margin = 2)
round(tablecaso1*100,2)
cat('\n')
cat('Grupo CTRL')
tablectrl <- table(CTRL20$PREMATCAT, CTRL20$PRENATAL)
tablectrl1 <- prop.table(tablectrl, margin = 2)
round(tablectrl1*100,2)

Grupo CASO

          
             nao   sim
  postermo  0.00  0.00
  ausente   0.00  0.00
  moderada 20.00 32.53
  alta     80.00 56.63
  extrema   0.00 10.84


Grupo CTRL

          
             nao   sim
  postermo 14.29 19.91
  ausente   7.14  5.58
  moderada 78.57 70.29
  alta      0.00  2.71
  extrema   0.00  1.51

Observando as frequências em mulheres com idade menor ou igual a 20 anos com relação as colunas da tabela (prenatal), verifica-se que no grupo CASO a maioria das ocorrências observadas são de prematuridade moderada e alta, com valores bem próximos entre as mães que fizeram o prenatal e as que não fizeram.
Já no grupo CONTROLE podemos observar que a maior ocorrência foi de prematuridade moderada, seguida pelo postermo, além disso vemos que os valores são bem próximos entre as mães que fizeram acompanhamento prenatal e as que não fizeram.

###6. Mulheres acima de 20 anos e que não fizeram um prenatal apresentem maiores chances de terem bebês prematuros.

In [ ]:
# Tabela da frequência relativa avaliando mulheres com idade maior que 20 e prenatal

CASO20 <- CASO[(CASO[,'IDADE'] > 20),]
CTRL20 <- CTRL[(CTRL[,'IDADE'] > 20),]

# Frequência calculadas nas COLUNAS
cat('Grupo CASO')
tablecaso <- table(CASO20$PREMATCAT, CASO20$PRENATAL)
tablecaso1 <- prop.table(tablecaso, margin = 2)
round(tablecaso1*100,2)
cat('\n')
cat('Grupo CTRL')
tablectrl <- table(CTRL20$PREMATCAT, CTRL20$PRENATAL)
tablectrl1 <- prop.table(tablectrl, margin = 2)
round(tablectrl1*100,2)

Grupo CASO

          
             nao   sim
  postermo  0.00  0.00
  ausente   0.00  0.00
  moderada 25.00 38.14
  alta     62.50 50.00
  extrema  12.50 11.86


Grupo CTRL

          
             nao   sim
  postermo 23.68 16.71
  ausente   2.63  4.77
  moderada 68.42 75.77
  alta      2.63  2.07
  extrema   2.63  0.68

Observando as frequências em mulheres com mais de 20 anos com relação as colunas da tabela (prenatal), verifica-se que no grupo caso a maioria dos ocorrências são de prematuridade alta. Já no grupo controle a maioria tanto como sim como não são de prematurudade moderada. Repetindo o mesmo resultado encontrado na analise anterior.

### Mulheres de etnias negras ou hispânicas que não fizeram o prenatal apresentem maiores chances de apresentarem filhos prematuros.

In [ ]:
# Tabela da frequência relativa avaliando mulheres que fizeram o acompanhamento prenatal e a suas etnias

CASOpre <- CASO[(CASO[,'PRENATAL'] == 'sim'),]
CTRLpre <- CTRL[(CTRL[,'PRENATAL'] == 'sim'),]

# Frequência calculadas nas linhas
cat('Grupo CASO')
tablecaso <- table(CASOpre$PREMATCAT, CASOpre$ETNIA)
tablecaso1 <- prop.table(tablecaso, margin = 1)
round(tablecaso1*100,2)
cat('\n')
cat('Grupo CTRL')
tablectrl <- table(CTRLpre$PREMATCAT, CTRLpre$ETNIA)
tablectrl1 <- prop.table(tablectrl, margin = 1)
round(tablectrl1*100,2)

Grupo CASO

          
           branca hispanica negra
  postermo                       
  ausente                        
  moderada  36.63      5.94 57.43
  alta      47.92     11.81 40.28
  extrema   34.38      9.38 56.25


Grupo CTRL

          
           branca hispanica negra
  postermo  56.47     18.68 24.86
  ausente   41.92     22.22 35.86
  moderada  49.40     21.59 29.00
  alta      68.18     18.18 13.64
  extrema   39.39     27.27 33.33

Analisando a prematuridade das mulheres que fizeram o acompanhamento prenatal em relação a etnia de cada uma, podemos observar que no grupo CASO os casos se concentraram em prematuridade moderada, alta e extrema, ocorrendo em sua grande maioria em mulheres de etnias branca e negra.
Já no grupo CONTROLE podemos observar que em todos os níveis de prematuridade, a maior ocorrência foi em mulheres de etnia branca, além disso também observamos que as ocorrências em mulheres de etnias negra e hispanica tiveram um valor bem próximo entre si.

In [ ]:
# Frequência calculadas nas colunas
cat('Grupo CASO')
tablecaso <- table(CASOpre$PREMATCAT, CASOpre$ETNIA)
tablecaso1 <- prop.table(tablecaso, margin = 2)
round(tablecaso1*100,2)
cat('\n')
cat('Grupo CTRL')
tablectrl <- table(CTRLpre$PREMATCAT, CTRLpre$ETNIA)
tablectrl1 <- prop.table(tablectrl, margin = 2)
round(tablectrl1*100,2)

Grupo CASO

          
           branca hispanica negra
  postermo   0.00      0.00  0.00
  ausente    0.00      0.00  0.00
  moderada  31.62     23.08 43.28
  alta      58.97     65.38 43.28
  extrema    9.40     11.54 13.43


Grupo CTRL

          
           branca hispanica negra
  postermo  19.24     15.26 15.12
  ausente    4.06      5.16  6.21
  moderada  73.13     76.64 76.66
  alta       2.94      1.88  1.05
  extrema    0.64      1.06  0.96

Observando agora a frequência em relação a coluna da tabela (etnia), percebemos que no grupo CASO a ocorrência de prematuridade alta foi a maior em todos as etnias das mulheres (branca, hispanica e negra), tendo a prematuridade moderada em segundo lugar.
Já no grupo CONTROLE, observamos que a maior ocorrÊncia nas etnias foi de prematuridade moderada, tendo inclusive valores bem próximos entre si em cada etnia, e segundo lugar tem o postermo, porém com um valor bem distante da moderada, mas bem parecidos entre si de uma etnia para outra.

### Mulheres acima de 20 anos e o RCN seja primogenito tenham maiores chances de terem um bebê prematuro.

In [ ]:
# Tabela da frequência relativa avaliando mulheres com idade maior que 20 e prenatal

CASO20 <- CASO[(CASO[,'IDADE'] > 20),]
CTRL20 <- CTRL[(CTRL[,'IDADE'] > 20),]

# Frequência calculadas nas LINHAS
cat('Grupo CASO')
tablecaso <- table(CASO20$PREMATCAT, CASO20$PRIMOG)
tablecaso1 <- prop.table(tablecaso, margin = 1)
round(tablecaso1*100,2)
cat('\n')
cat('Grupo CTRL')
tablectrl <- table(CTRL20$PREMATCAT, CTRL20$PRIMOG)
tablectrl1 <- prop.table(tablectrl, margin = 1)
round(tablectrl1*100,2)

Grupo CASO

          
             nao   sim
  postermo            
  ausente             
  moderada 60.53 39.47
  alta     63.73 36.27
  extrema  58.33 41.67


Grupo CTRL

          
             nao   sim
  postermo 58.64 41.36
  ausente  64.81 35.19
  moderada 63.85 36.15
  alta     59.15 40.85
  extrema  79.17 20.83

Ao analisar essa hipótese pela frequência vimos que essa afirmação não é verdadeira, pois o número de mulheres que tiveram filhos com um grau de prematuridade tanto no grupo caso, quanto no grupo controle, foi maior quando não existiu a primogenidade.

In [ ]:
# Frequência calculadas nas colunas
cat('Grupo CASO')
tablecaso <- table(CASO20$PREMATCAT, CASO20$PRIMOG)
tablecaso1 <- prop.table(tablecaso, margin = 2)
round(tablecaso1*100,2)
cat('\n')
cat('Grupo CTRL')
tablectrl <- table(CTRL20$PREMATCAT, CTRL20$PRIMOG)
tablectrl1 <- prop.table(tablectrl, margin = 2)
round(tablectrl1*100,2)

Grupo CASO

          
             nao   sim
  postermo  0.00  0.00
  ausente   0.00  0.00
  moderada 36.80 38.96
  alta     52.00 48.05
  extrema  11.20 12.99


Grupo CTRL

          
             nao   sim
  postermo 15.61 18.78
  ausente   4.88  4.52
  moderada 76.67 74.01
  alta      1.95  2.30
  extrema   0.88  0.40

No grupo caso, foi observado uma maio concentração de dados na prematuridade alta, tanto para bebês primogênitos ou não, já no grupo controle, a maior concentração de dados foi no prematuridade moderada.

### Pessoas de rendas baixas, apresentem maiores chances de terem bebês prematuros e de baixo peso.

In [ ]:
# Tabela da frequência relativa avaliando bebês com peso baixo e renda
CASOb <- CASO[(CASO[,'pesoCAT'] == 'baixo'),]
CTRLb <- CTRL[(CTRL[,'pesoCAT'] == 'baixo'),]

# Frequência calculadas nas linhas
cat('Grupo CASO')
tablecaso <- table(CASOb$PREMATCAT, CASOb$rendaCAT)
tablecaso1 <- prop.table(tablecaso, margin = 1)
round(tablecaso1*100,2)
cat('\n')
cat('Grupo CTRL')
tablectrl <- table(CTRLb$PREMATCAT, CTRLb$rendaCAT)
tablectrl1 <- prop.table(tablectrl, margin = 1)
round(tablectrl1*100,2)

Grupo CASO

          
            baixa  media   alta
  postermo                     
  ausente                      
  moderada  96.15   3.85   0.00
  alta      96.08   3.27   0.65
  extrema  100.00   0.00   0.00


Grupo CTRL

          
            baixa  media   alta
  postermo  91.67   0.00   8.33
  ausente  100.00   0.00   0.00
  moderada  90.40   8.59   1.01
  alta      88.33  11.67   0.00
  extrema   96.30   3.70   0.00

Analisando as frequencias por linhas, pode ser observado que no Grupo Caso, as amostras que tiveram alguma prematuridade, em sua grande maioria eram mulheres que tinham uma renda baixa, ja no grupo controle, pode tambem ser observado o mesmo comportamento para as mulheres que tiveram filhos com algum grau de prematuridade, e o RCN apresentou peso baixo, sendo em grande parte concentrado em mulheres de renda baixa.  

In [ ]:
# Frequência calculadas nas linhas
cat('Grupo CASO')
tablecaso <- table(CASOb$PREMATCAT, CASOb$rendaCAT)
tablecaso1 <- prop.table(tablecaso, margin = 2)
round(tablecaso1*100,2)
cat('\n')
cat('Grupo CTRL')
tablectrl <- table(CTRLb$PREMATCAT, CTRLb$rendaCAT)
tablectrl1 <- prop.table(tablectrl, margin = 2)
round(tablectrl1*100,2)

Grupo CASO

          
            baixa  media   alta
  postermo   0.00   0.00   0.00
  ausente    0.00   0.00   0.00
  moderada  35.71  44.44   0.00
  alta      52.50  55.56 100.00
  extrema   11.79   0.00   0.00


Grupo CTRL

          
           baixa media  alta
  postermo  4.04  0.00 33.33
  ausente   1.10  0.00  0.00
  moderada 65.81 68.00 66.67
  alta     19.49 28.00  0.00
  extrema   9.56  4.00  0.00

Obervando a frequência de prematuridade por coluna, pode se observar que no grupo CASO as ocorrências se concentram em moderada, alta e extrema para famílias de baixa renda, em moderada e alta para famílias de renda média e em alta para famílias de alta renda.
Já no grupo CONTROLE foi observado que, para famílias de baixa renda, a concentração maior ficou em moderada e alta, para famílias de renda média a concentração ficou em moderada e alta e para famílias de alta renda a concentração ficou em moderada e postermo.

# 4.   Conclusão



Com base nas análises realizadas, podemos chegar a algumas conclusões:

Para análise de Outiliers:
Outliers de idade e prematuridade: Foi identificado um dado atípico em relação à idade de uma mãe no grupo CASO, que estava acima dos 40 anos. No entanto, é importante considerar que essa mãe seguiu as práticas adequadas durante a gravidez (não fez uso de cigarro, álcool e realizou o pré-natal). Além disso, alguns casos extremos de prematuridade foram observados, sendo necessário verificar possíveis doenças relacionadas ao parto ou erros de digitação.

Outliers de peso: No grupo CONTROLE, foram encontradas 78 observações atípicas em relação ao peso, enquanto no grupo CASO não foram encontradas outliers de peso. Essa diferença pode ser relevante para a análise posterior. Temos ainda 285 observações no tratamento controle, que se encaixam no tratamento CASO.

Prematuridade alta e extrema e impacto na hospitalização: Valores extremos de prematuridade estiveram associados a longos períodos de hospitalização, especialmente nos casos mais graves. Também foi observado que bebês que não receberam tratamento adequado do governo (devido ao peso abaixo de 2500 kg) apresentaram maior impacto na saúde e desenvolvimento. Foi encontrado um possível outlier onde, a prematuridade foi de 14 semanas (extrema) e passou 90 semanas hospitalizado. Ou pode ser um erro de digitação ou alguma doença apresentada na hora do nascimento.

Em relação as hipóteses levantadas, pode-se chegar a algumas conclusões:

Conclusões para o Grupo CASO:

Etnia e prematuridade: Mulheres com menos de 20 anos no Grupo CASO apresentaram maior incidência de prematuridade nas categorias moderada, alta e extrema, sendo que a maioria dos casos ocorreu em mulheres negras.

Uso de cigarro e álcool: Mulheres que fumaram durante a gravidez e tiveram prematuridade moderada apresentaram bebês com QI baixo, enquanto aquelas que tiveram prematuridade alta e extrema apresentaram bebês com QI médio.

Acompanhamento pré-natal: A maior ocorrência de prematuridade moderada e alta foi observada em mulheres com menos de 20 anos, independentemente de terem feito o acompanhamento pré-natal.

Etnia e prematuridade em mulheres com mais de 20 anos: No Grupo CASO, mulheres com mais de 20 anos, independentemente de serem negras ou brancas, apresentaram maior incidência de prematuridade alta, e a maior frequência de prematuridade extrema foi observada em mulheres negras.

Etnia e prematuridade em mulheres com mais de 20 anos e com relação ao acompanhamento pré-natal: Mulheres do Grupo CASO com mais de 20 anos, independentemente de terem feito ou não o acompanhamento pré-natal, apresentaram maior incidência de prematuridade alta.

Renda e prematuridade: No Grupo CASO, a maioria das amostras que tiveram prematuridade, nas categorias moderada, alta e extrema, pertencia a famílias de baixa renda.

Conclusões para o Grupo CONTROLE:

Etnia e prematuridade: Mulheres com menos de 20 anos no Grupo CONTROLE apresentaram maior incidência de prematuridade moderada e alta, sendo que a maior frequência de prematuridade extrema ocorreu em mulheres negras.

Uso de cigarro e álcool: No Grupo CONTROLE, a prematuridade extrema esteve associada a bebês com QI baixo, independentemente do uso de cigarro e álcool durante a gravidez.

Acompanhamento pré-natal: A maior ocorrência de prematuridade moderada foi observada em mulheres com menos de 20 anos, tanto na presença quanto na ausência de acompanhamento pré-natal.

Etnia e prematuridade em mulheres com mais de 20 anos: Mulheres com mais de 20 anos no Grupo CONTROLE, independentemente de serem negras ou brancas, apresentaram maior incidência de prematuridade moderada, e a prematuridade extrema foi mais frequente em mulheres negras.

Etnia e prematuridade em mulheres com mais de 20 anos e com relação ao acompanhamento pré-natal: No Grupo CONTROLE, mulheres com mais de 20 anos, que fizeram ou não o acompanhamento pré-natal, apresentaram maior incidência de prematuridade moderada.

Renda e prematuridade: No Grupo CONTROLE, a maioria das amostras que tiveram prematuridade moderada e alta pertencia a famílias de baixa renda, enquanto a prematuridade extrema também foi mais frequente em famílias de baixa renda.

Essas conclusões destacam a importância de considerar a etnia, a idade e a renda das mulheres no contexto da prematuridade, bem como os possíveis efeitos do uso de cigarro e álcool durante a gravidez e o acompanhamento pré-natal. No entanto, é essencial lembrar que essas conclusões são baseadas em análises preliminares e, para obter conclusões mais robustas e precisas, é fundamental realizar estudos estatísticos mais aprofundados, controlando outros possíveis fatores de confusão e ampliando a amostra para uma análise mais abrangente.










# 5.   Referências





Almeida AC, Jesus ACP, Lima PFT, Araújo MFM, Araújo TM. Fatores de risco maternos para prematuridade em uma maternidade pública de Imperatriz-MA. Rev Gaúcha Enferm. 2012. 33(2):86-94. Disponível em:https://www.scielo.br/j/rgenf/a/xd37QdPqHsLPcy5WdB3QKzg/?format=pdf&lang=pt

Bernardino SSS. Estado nutricional de recém-nascidos pré-termo internados em unidades neonatais. Currais Novos-RN: Universidade Federal do Rio Grande do Norte; 2019.

Bertani, A. L., Garcia, T., Tanni, S. E. & Godoy, I. (2015). Prevenção do tabagismo na gravidez: importância do conhecimento materno sobre os malefícios para a saúde e opções de tratamento disponíveis. Revista Jornal Brasileiro Pneumologia, 41(2), 175-181. http://dx.doi.org/10.1590/S1806- 37132015000004482.

 Blencowe H, Cousens S, Oestergaard MZ, Chou D, Moller AB, Narwal R et al.National, regional, and worldwide estimates of preterm birth rates in the year 2010 with time trends since 1990 for selected countries: a systematic analysis and implications.
Lancet. 2012; 379(9832). Disponível em: https://doi.org/10.1016/S0140-6736(12)60820-4.

Cabral, R. A., Santos, B. M. O. & Cano, M. A. T. (2017). A experiência de ser mãe pela primeira vez após os 35 anos. Revista Cinergis, 18(4), 279-284. http://dx.doi.org/10.17058/cinergis.v18i4.9830.

Ferreira Junior, A. R., Albuquerque, R. A. S., Aragão, S. R. & Rodrigues, M. E. N. G. (2018). Perfil epidemiológico de mães e recém-nascidos prematuros. Revista Enfermagem Contemporânea, 7(1), 6-12. http://10.17267/2317-3378rec.v7i1.1159.

Kawakita, T., Wilson, K., Grantz, K. L., Landy, H. J., Huang, C. C. & Go/mez-Lobo, V. (2016). Adverse maternal and neonatal outcomes in adolescent pregnancy. J Pediatr Adolesc Gynecol, 29(2), 130-136. http://10.1016/j.jpag.2015.08.006.

Oliveira LL, Gonçalves AC, Costa JSD, Bonilha ALL. Fatores maternos e neonatais relacionados à prematuridade. Rev Esc Enferm USP. 2016. 50(3):382-389. Disponível em:
https://www.scielo.br/j/reeusp/a/7MGKxJcY8Ldgf8ynN69LWJk/?format=pdf&lan
g=pt

Ramos, H. Â. de C., & Cuman, R. K. N.. (2009). Fatores de risco para prematuridade: pesquisa documental. Escola Anna Nery, 13(2), 297–304. https://doi.org/10.1590/S1414-81452009000200009

Ross GS. Home intervention for premature infants of low-income families. American Journal of Orthopsychiatry 1984;54(2):263-270.

Silva, L. S. R., Cavalcante, A. N., Carneiro, J. K. R. & Oliveira, M. A. S. (2020). Índice de Apgar correlacionado a fatores maternos, obstétricos e neonatais a partir de dados coletados no Centro de Saúde da Família do bairro Dom Expedito Lopes situado no município de Sobral/CE. Revista Cienfica da FMC, 15(1), 25-30. http://10.29184/1980-7813.rcfmc.

Sociedade de Pediatria de São Paulo (2017). Efeitos do álcool na gestante, no feto e no recém-nascido. São Paulo. Retrieved from https://www.spsp.org.br/downloads/AlcoolSAF2.pdf.

World Health Organization (WHO). Preterm birth. Geneva: WHO; 2018. Disponível em: http://www.who.int/mediacentre/factsheets/fs363/en/
